src/generic-container.test.ts

```diff
@@ -74,6 +74,30 @@ describe("GenericContainer", () => {
     await expect(fetch(url)).rejects.toThrowError();
   });
 
+  it("should build and start from a Dockerfile with build arguments", async () => {
+    const context = path.resolve(__dirname, "..", "docker-with-buildargs");
+    const container = await GenericContainer.fromDockerfile(context, undefined, undefined, {
```

Comment:
Not a fan of these `undefined`. If these arguments have to be there it'd be better to have `fromDockerfile` accept an object of parameters, so that we can skip the optional ones.

Reply:
Like I said, I'll do a rework for the arguments ;)